python/ray/tune/tests/_test_multi_tenancy_run.py

```diff
@@ -85,3 +86,8 @@ def train_func(config):
 # Put assertions last, so we don't finish early because of failures
 assert sorted([result.metrics["param"] for result in results]) == VALS
 assert [result.metrics["fixed"] for result in results] == [FIXED_VAL, FIXED_VAL]
+
+if __name__ == "__main__":
+    import pytest
+
+    sys.exit(pytest.main(["-v", __file__]))
```

Comment:
This is changed by the bazel linter?

Reply:
linter complains no main function, so I added it

Reply:
I checked the code, it's intended. This itself is not a pytest, it's used in another test file `test_multi_tenancy.py`.

Reply:
gotcha, reverted.